src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -21,14 +21,14 @@
 import seedu.address.model.patient.Patient;
 
 /**
- * Adds a patient to the address book.
+ * Adds a patient to the patient record.
```

Comment:
Perhaps this could be clearer since patient record could refer to the Record (the one with observations, diagnosis, treatment plan)? Maybe to patient list or Advanced&Efficient?